_includes/layout/base/wallets-selector.html

```diff
@@ -0,0 +1,601 @@
+            {% for wallet in site.wallets %}
+              {% for platform in wallet.platform %}
+                {% for os in platform.os %}
+                  <a
+                    id="walletLink"
+                    href="{% if platform.name == os.name %}/{{page.lang}}/wallets/{{platform.name}}/{{wallet.id}}/{% else %}/{{page.lang}}/wallets/{{platform.name}}/{{os.name}}/{{wallet.id}}/{% endif %}"
+                    class="wallet-table-row wallet-link"
+                    data-categories="{% for check in os.check %}{% assign checkValue = check[1] %}{% assign checkName = check[0] %}{% if checkValue contains 'checkgood' %}{{checkName}} {% endif %}{% endfor %}{% if os.privacycheck %}{% for privacycheck in os.privacycheck %}{% assign privacycheckValue = privacycheck[1] %}{% assign privacycheckName = privacycheck[0] %}{% if privacycheckValue contains 'checkgood' or privacycheckValue contains 'checkpass' %}{{privacycheckName}} {% endif %}{% endfor %}{% endif %}{{os.name}}"
+                  >
+                    <div class="wallet-table-data">
+                      <img class="wallet-img" src="/img/wallet/{{ wallet.id }}.png?{{site.time | date: '%s'}}" alt="{{ wallet.title }}" />
+                      <span class="wallet-label">{{wallet.title}}</span>
+                    </div>
+                    {% for check in os.check %}
+                      {% assign checkName = check[0] %}
+                      {% if checkName == "privacy" %}
+                        {% continue %}
+                      {% endif %}
+                      {% assign checkValue = check[1] %}
+                      <div class="wallet-table-data">
+                        <div class="wallet-{% if checkValue contains 'checkgood' %}good{% elsif checkValue contains 'checkpass' %}good{% elsif checkValue contains 'checkfail' %}caution{% elsif checkValue contains 'checkneutral' %}neutral{% endif %}"></div>
+                      </div>
+                    {% endfor %}
+                    {% if os.privacycheck %}
+                      {% for privacycheck in os.privacycheck %}
+                        {% assign privacycheckValue = privacycheck[1] %}
+                        <div class="wallet-table-data">
+                          <div class="wallet-{% if privacycheckValue contains 'checkgood' %}good{% elsif privacycheckValue contains 'checkpass' %}good{% elsif privacycheckValue contains 'checkfail' %}caution{% elsif privacycheckValue contains 'checkneutral' %}neutral{% endif %}"></div>
+                        </div>
+                      {% endfor %}
+                    {% else %}
+                      <div class="wallet-table-data">-</div>
+                      <div class="wallet-table-data">-</div>
+                      <div class="wallet-table-data">-</div>
+                    {% endif %}
+                  </a>
+                {% endfor %}
+              {% endfor %}
+            {% endfor %}
+          </div>
+          <div class="wallet-hints-row">
```

Comment:
Let's not show the hints until an operating system has been chosen.

Reply:
Are they visible? I hid them by default. They are visible only if the operating system is selected and at least one wallet matches the filter values.

Reply:
Apologies, you're right - I wasn't looking at an updated preview. Thanks for taking care of this.